__std__/nix/doc/library/default.nix

```diff
@@ -0,0 +1,10 @@
+{ inputs, cell }@organelle:
```

Comment:
What is this `@organelle` syntax? Is that a Nix, flakes or std thing?

Reply:
Nix thing. Syntax sugar essentially.
```
{ inputs, cell }:
{
  common = import ./common.nix { inherit inputs cell; };
  haskell = import ./haskell.nix { inherit inputs cell; };
}
```
Becomes:
```
{ inputs, cell }@anything-really:
{
  common = import ./common.nix anything-really;
  haskell = import ./haskell.nix anything-really;
}
```